Migration script/openIMIS migration latest.sql

```diff
@@ -4122,31 +4249,25 @@ Go
 
 
 -- OTC-144: Set isOffline status to 0 for insurees in database 
-UPDATE tblInsuree SET isOffline=0 
-	WHERE (SELECT 1 FROM tblIMISDefaults where OfflineCHF = 1 And OfflineHF = 1) is null;
-
-UPDATE tblFamilies SET isOffline=0 
-	WHERE (SELECT 1 FROM tblIMISDefaults where OfflineCHF = 1 And OfflineHF = 1) is null;
-
-UPDATE tblInsureePolicy SET isOffline=0 
-	WHERE (SELECT 1 FROM tblIMISDefaults where OfflineCHF = 1 And OfflineHF = 1) is null;
-
-UPDATE tblPremium SET isOffline=0
-	WHERE (SELECT 1 FROM tblIMISDefaults where OfflineCHF = 1 And OfflineHF = 1) is null;
-
-UPDATE tblPolicy SET isOffline=0
-	WHERE (SELECT 1 FROM tblIMISDefaults where OfflineCHF = 1 And OfflineHF = 1) is null;
-
+IF NOT EXISTS (SELECT 1 FROM tblIMISDefaults where OfflineCHF = 1 OR OfflineHF = 1)
+BEGIN
+	UPDATE tblInsuree SET isOffline=0 
+	UPDATE tblFamilies SET isOffline=0 
+	UPDATE tblInsureePolicy SET isOffline=0 
+	UPDATE tblPremium SET isOffline=0
+	UPDATE tblPolicy SET isOffline=0
+END 
 
 -- ready status support
+IF OBJECT_ID('uspMatchPayment', 'P') IS NOT NULL
+    DROP PROCEDURE uspMatchPayment
+GO
 
-
-/****** Object:  StoredProcedure [dbo].[uspMatchPayment]    Script Date: 11/4/2020 3:19:16 PM ******/
 SET ANSI_NULLS ON
 GO
 SET QUOTED_IDENTIFIER ON
 GO
-ALTER PROCEDURE [dbo].[uspMatchPayment]
+CREATE PROCEDURE [dbo].[uspMatchPayment]
```

Comment:
Miss the delete is exisitng

Reply:
I was fixing the conflicts. Please check now.